doc/articles/dummies1.md

```diff
@@ -0,0 +1,146 @@
+# How to migrate existing UWP code to Uno
+
+There are two separate paths for using an existing UWP codebase on top of Uno:
+- An existing UWP application
+- An existing UWP library
+
+For migrating a UWP library, see [this article](howto-migrate-existing-code.md).
+
+# How to migrate UWP app to Uno?
+# Step by step guide.
+
+This guide assumes that you have a UWP app, and want to convert this app to be multi-platform.
+
+
+
+In short:
+* convert solution to use Uno,
+* check if any APIs you use are not implemented in Uno.
+* if so, you have two options:
+	* try to find a workaround for this, or
+	* extend Uno
+* launch and test the converted app.
+
+
+## create Uno project
+You have two options:
+
+### 1. Adding new projects to existing solution
+First, open your app solution in Visual Studio. Then, using the Solution Explorer window:
+* Add new projects (right click on Solution, Add, New Project, Visual C#, Uno Plaform, Cross-Plaform App (Uno Platform), using same name as your existing project (APPNAME) with "\_Uno" suffix. In effect, APPNAME_Uno folder will be created, and inside it, APPNAME_Uno.Shared folder, and four 'heads', one for each target platform (as APPNAME_Uno.Droid, etc.).
+* while reading next parts of this guide, treat `APPNAME[_Uno]` as `APPNAME_Uno`.
+
+### 2. Creating a new solution
+* Create a new solution with the Uno app template for Visual Studio, using the same name as your existing project (APPNAME). In effect, APPNAME folder will be created, and inside it, APPNAME.Shared folder, and four 'heads', one for each target platform (as APPNAME.Droid, etc.).
+* while reading next parts of this guide, treat `APPNAME[_Uno]` as `APPNAME`.
+
+### tips for both options above
+* description of Uno app solution structure is [documented here](uno-app-solution-structure.html).
```

Comment:
```suggestion
* description of Uno app solution structure is [documented here](uno-app-solution-structure.md).
```

Reply:
Are you sure? doc/articles/supported-features.md , under 'Design fidelity' caption, link 'Path & Shapes support' is with .md, not .html  (same for all other links).

Reply:
Yes, all links to other articles should be in the form `foo.md`, that's the change I'm suggesting. :)

The `docfx` tool will automatically convert these when it generates the `platform.uno` docs.